test/cases/testdata/reachable/test-reachable-paths-0422.yaml

```diff
@@ -0,0 +1,170 @@
+cases:
+  - data: {}
+    input_term: '{"a"}'
```

Comment:
I don't believe it's material that this set comes from input, so we could do 
```
        package reachable
        
        p = result {
          graph.reachable_paths({}, {"a"}, result)
        }
```

but it's OK and doesn't hurt to do it like this.